cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/ResponseUtils.java

```diff
@@ -101,6 +122,6 @@ public static void writeErrorResponse(HttpServletResponse response,
     } else {
       responseMsg = errorMessage == null ? "" : errorMessage;
     }
-    writeResponseToOutputStream(response, responseCode, json, responseMsg);
+    writeResponseToOutputStream(response, responseCode, json, responseMsg, null);
```

Comment:
Calling this function with a `null` config, which prevents us from using `cors` header properly.

Reply:
reason is mentioned in previous comment